index.html

```diff
@@ -32,85 +32,102 @@
-            contact me.
-          </p>
-          <div class="social-icons">
-            <a href="" id="connect">LET'S CONNECT</a>
-            <ul id="icons">
-              <li>
-                <a href="#">
-                  <img src="./images/twitter.svg" alt="twitter-logo" />
-                </a>
-              </li>
-              <li>
-                <a href="#"
-                  ><img src="./images/Linkedin icon.svg" alt="linkedin-logo"
-                /></a>
-              </li>
-              <li>
-                <a href="#"
-                  ><img src="./images/medium.svg" alt="medium-logo"
-                /></a>
-              </li>
-              <li>
-                <a href="#"
-                  ><img src="./images/github.svg" alt="github-logo"
-                /></a>
-              </li>
-              <li>
-                <a href="#"
-                  ><img src="./images/angelist.svg" alt="angelist-logo"
-                /></a>
-              </li>
-            </ul>
+        <div class="container">
+          <div class="main-text">
+            <h1>
+              I’m Jeysson. <br />
+              Glad to see you!
+            </h1>
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+            <div class="social-icons social-deskt">
+              <a href="#" class="connect conect-deskt">LET'S CONNECT</a>
+              <ul id="icons">
+                <li>
+                  <a href="#">
+                    <img src="./images/twitter.svg" alt="twitter-logo" />
+                  </a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="./images/Linkedin icon.svg" alt="linkedin-logo"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="./images/medium.svg" alt="medium-logo"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="./images/github.svg" alt="github-logo"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="./images/angelist.svg" alt="angelist-logo"
+                  /></a>
+                </li>
+              </ul>
+            </div>
           </div>
         </div>
       </section>
 
-      <section id="works">
-        <div class="card">
-          <div class="head-image">
-            <img
-              src="./images/Snapshoot Portfolio.png"
-              alt="work-image-project"
-            />
-            <h2 class="tonic">TONIC</h2>
-            <ul class="dates">
+      <section class="works" id="works">
```

Comment:
- [ ] [OPTIONAL] Hi, nice work on the css media queries. I noticed that your media queries don’t work well for devices between 768px to 1300px because for these widths, the projects description comes outside its container. You might want to adjust the media queries to make sure the projects description stays inside the container for all device widths. This will improve the responsiveness and user experience of your page.

<img width="871" alt="image" src="https://github.com/alyconr/Personal-Portfolio-Project/assets/52934128/733ef436-722b-43a5-a0f9-9fff7b81b692">